lib/src/widgets/chat/ds_text_message_bubble.widget.dart

```diff
@@ -43,11 +43,14 @@ class _DSTextMessageBubbleState extends State<DSTextMessageBubble> {
   late final bool _isDefaultBubbleColors;
   late final bool _isLightBubbleBackground;
 
-  _DSTextMessageBubbleState() {
+  @override
+  void initState() {
     _isDefaultBubbleColors =
         widget.style.isDefaultBubbleBackground(widget.align);
     _isLightBubbleBackground =
         widget.style.isLightBubbleBackground(widget.align);
+
+    super.initState();
```

Comment:
As described in the Flutter doc, the initState method, if override, should **start** with the ```super.initState``` call.

> Implementations of this method should start with a call to the inherited method, as in super.initState().

Reference: https://api.flutter.dev/flutter/widgets/State/initState.html

Reply:
Done.